src/backend/binaryen/utils.ts

```diff
@@ -771,6 +785,25 @@ export namespace FunctionalFuncs {
         return binaryen.none;
     }
 
+    export function unboxAnyToAnyTypedExtref(
```

Comment:
why it's named as "AnyTypedExtref"?

Reply:
Our table is any type, in normal situation, we need get elem from the table, then cast to its real type. But here, I just want to get the any-typed element, no casting.

Reply:
unboxAnyToExtrefWithoutCast